pkg/naiserator/config/config.go

```diff
@@ -78,7 +78,7 @@ const (
 	SecurelogsConfigMapReloadImage      = "securelogs.configmap-reload-image"
 	SecurelogsFluentdImage              = "securelogs.fluentd-image"
 	VaultAddress                        = "vault.address"
-	VaultAuthPath                       = "vault.auth-path"
+	VaultAuthPath                       = "vault.auth-path-new"
```

Comment:
Kunne vi brukt `v2` i stedet for `new`. For hva gjør vi hvis vi ønsker å endre denne igjen?

Reply:
Se forrige svar.